bruno/collections/Rafiki/scripts.js

```diff
@@ -106,7 +106,7 @@ const scripts = {
     return `t=${timestamp}, v${version}=${digest}`
   },
 
-  addApiSignatureHeader: function (packageName) {
+  addApiSignatureHeader: function (packageName, instance) {
```

Comment:
In `loadWalletAddressIdsIntoVariables`, we also need to pass in `tenant-id` header. Otherwise, the Examples > Admin API requests don't work where we use `loadWalletAddressIdsIntoVariables`